Tests/DatabaseServiceTests/Repositories/UserRepositoryDeleteTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using DataBaseService.Database;
+using DataBaseService.Database.Models;
+using DataBaseService.Mappers;
+using DataBaseService.Repositories;
+using Kernel.CustomExceptions;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using System.Linq;
+
+namespace DatabaseServiceTests
+{
+    public class UserRepositoryDeleteTests
+    {
+        UserRepository userRepository;
+        DbUserCredential dbUserCredential;
+        UserMapper mapper;
+        Guid userId;
+        Guid id;
+
+        [SetUp]
```

Comment:
I'm not sure, but may be there's [OneTimeSetUp] and it might be better here?
Do yoou need to re-initialize it before each test?

Reply:
Yes, I don't need to initialize this every time

Reply:
Until this moment, I did not know about this attribute, now I will correct it

Reply:
Hmm
Strnage, but with [OneTimeSetUp] it doesn`t work(